tests/console/consoletest_setup.pm

```diff
@@ -32,6 +34,20 @@ sub run {
     save_screenshot;
     check_var("BACKEND", "ipmi") ? use_ssh_serial_console : select_console 'root-console';
 
+    # adding sdk repositories, needed by some tests later
+    add_suseconnect_product('sle-module-desktop-applications') if is_sle("15-sp1+");
```

Comment:
This change is wrong for several reasons:
1. The function `disable_bash_mail_notification` is the wrong place to register a module.
2. `tests/console/consoletest_setup.pm` will register the module for all scenarios and will cause false negatives.
3. If sssd needs SDK module, then the test module sssd has to register and also unregister the module, so that following modules are not affected by a system with registered SDK where no SDK should be.